scheduler/scheduler.go

```diff
@@ -228,7 +234,8 @@ func Create(depGraph DependencyGraph) {
 	//TODO Make sure every KO gets created eventually
 }
 
-//Kosaraju's algorithm implementation https://en.wikipedia.org/wiki/Kosaraju%27s_algorithm
+//DetectCycles implements Kosaraju's algorithm https://en.wikipedia.org/wiki/Kosaraju%27s_algorithm
+//for dedecting cycles in graph.
```

Comment:
s/dedecting/detecting
(although Kosaraju's algorithm isn't really a cycle detection algorithm at it's core).